PyDynamic/model_estimation/fit_filter.py

```diff
@@ -7,37 +7,26 @@
 
 This module contains the following functions:
 
-* :func:`LSIIR`: Least-squares IIR filter fit to a given frequency response
+* :func:`LSIIR`: Least-squares (time-discrete) IIR filter fit to a given frequency
+  response or its reciprocal optionally propagating uncertainties.
 * :func:`LSFIR`: Least-squares fit of a digital FIR filter to a given frequency
-  response
+  response.
 * :func:`invLSFIR`: Least-squares fit of a digital FIR filter to the reciprocal of a
   given frequency response.
 * :func:`invLSFIR_unc`: Design of FIR filter as fit to reciprocal of frequency response
   values with uncertainty
 * :func:`invLSFIR_uncMC`: Design of FIR filter as fit to reciprocal of frequency
   response values with uncertainty via Monte Carlo
-* :func:`invLSIIR`: Design of a stable IIR filter as fit to reciprocal of frequency
-  response values
-* :func:`invLSIIR_unc`: Design of a stable IIR filter as fit to reciprocal of frequency
-  response values with uncertainty
 
 """
-import warnings
+from typing import Optional, Tuple, Union
 
 import numpy as np
 import scipy.signal as dsp
 
-from ..misc.filterstuff import grpdelay, mapinside
+from ..misc.filterstuff import grpdelay, isstable, mapinside
 
-__all__ = [
-    "LSIIR",
-    "LSFIR",
-    "invLSFIR",
-    "invLSFIR_unc",
-    "invLSIIR",
-    "invLSIIR_unc",
-    "invLSFIR_uncMC"
-    ]
+__all__ = ["LSIIR", "LSFIR", "invLSFIR", "invLSFIR_unc", "invLSFIR_uncMC"]
```

Comment:
Do we want to reintroduce `invLSIIR` and `invLSIIR_unc` back into the `__all__`-string?

Reply:
I left them out to encourage the use of the new functions, which results in `import PyDynamic.model_estimation` not providing these functions, but only the new implementation. Would you rather want to have them?

Reply:
I was more worried about import behavior. As this does not seem to be a problem at all, I agree on your view. 👍